src/odemis/driver/hamamatsurx.py

```diff
@@ -555,13 +586,13 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
 
         # Set parameters streak unit
         parent.DevParamSet(self.location, "Time Range", "1 ns")
-        parent.DevParamSet(self.location, "MCP Gain", "0")
+        parent.DevParamSet(self.location, "MCP Gain", 0)
         # Switch Mode to "Focus", MCPGain = 0 (implemented in RemoteEx and also here in the driver).
         parent.DevParamSet(self.location, "Mode", "Focus")
         # Resets behavior for a vertical single shot sweep: Automatic reset occurs after each sweep.
         parent.DevParamSet(self.location, "Trig. Mode", "Cont")
         # [Volt] Input and indication of the trigger level for the vertical sweep.
-        parent.DevParamSet(self.location, "Trig. level", "1") # TODO??
+        parent.DevParamSet(self.location, "Trig. level", 1) # TODO??
```

Comment:
Explain the `TODO`. Do you mean that you wonder if it should be an argument, or a VA?